src/controllers/talos_pos_tracking.cpp

```diff
@@ -215,7 +219,7 @@ namespace inria_wbc {
 
         void TalosPosTracking::remove_contact(const std::string& contact_name)
         {
-            tsid_->removeRigidContact(contact_name);
+            assert(tsid_->removeRigidContact(contact_name));
```

Comment:
We should never do this because assert are removed when we compile with -DNDEBUG! (default for cmake in release).
=> never put any "useful" code in an assert.